data structures/java/JAVA Algorithms/Array/1d array example.txt

```diff
@@ -0,0 +1,20 @@
+// both are valid declarations
+int intArray[]; 
+or int[] intArray; 
+
+byte byteArray[];
+short shortsArray[];
+boolean booleanArray[];
+long longArray[];
+float floatArray[];
+double doubleArray[];
+char charArray[];
+
+// an array of references to objects of
+// the class MyClass (a class created by
+// user)
+MyClass myClassArray[]; 
+
+Object[]  ao,        // array of Object
+Collection[] ca;  // array of Collection
+                     // of unknown type
```

Comment:
Remove txt, docs, and documentation related files all over the PR.
Include Comments in files instead of these files.